client/landing/stepper/hooks/use-setup-onboarding-site.ts

```diff
@@ -119,9 +107,8 @@ export function useSetupOnboardingSite( options: SetupOnboardingSiteOptions ) {
 		if ( ( ignoreUrl || shouldSetupOnboardingSite() ) && site && flow ) {
 			return Promise.all( [
 				postSiteSettings( site, state ),
-				postSiteLogo( state ).then( ( logoUploadResult ) =>
-					setPattern( site, flow, logoUploadResult )
-				),
+				postSiteLogo( state ).then( () => Promise.resolve() ),
```

Comment:
```suggestion
				postSiteLogo( state ),
```

Reply:
The problem here is the return value of `postSiteLogo( state )` is not a Promise and doesn't match the argument type of `Promise.all` 😅

Reply:
I just double-checked and it does return a promise.

Reply:
After checking again, I found the issue is the value is non-void. So, I add the type definition to each promise as below
```diff
- Promise.all
+ Promise.all< void, SetSiteLogoResponse | void, void, void, void >
```